interface/patient_file/summary/stats.php

```diff
@@ -127,7 +127,7 @@ function load_location(location) {
     }
 
     if ($old_key == "medication" && $GLOBALS['erx_enable'] && $erx_upload_complete == 1) {
-        $display_current_medications_below = 1;
+        $display_current_medications_below = 0;
```

Comment:
went back into the original overhaul and thinking that this makes the most sense since without it the value is always `1`